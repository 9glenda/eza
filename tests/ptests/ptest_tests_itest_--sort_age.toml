bin.name = "eza"
args = "tests/itest --sort age"