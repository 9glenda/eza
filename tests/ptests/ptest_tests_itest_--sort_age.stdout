i
c
image.jpg.img.c.rs.log.png
d
e
l
p
q
b
f
o
j
k
n
a
m
h
g
vagrant
exa
index.svg
